### ISSUE TITLE
Tests are not rendered in folders in the desktop-gui on windows

### ISSUE THREAD

**Issue by corneliutusnea** [cypress-io/cypress#1335]
- Operating System: Windows 10
- Cypress Version: 2.0.1
- Browser Version: Chrome 63

### Is this a Feature or Bug?
Bug

### Current behavior:
Tests are incorrectly rendered in groups/folders on windows.
![image](https://user-images.githubusercontent.com/346349/36347487-2c12f922-14ac-11e8-9685-b593ff687423.png)

### Desired behavior:
Tests should be organized by groups

### How to reproduce:
Load a folder with tests organized in sub-folders on a Windows PC.

**Comment by corneliutusnea** [comment]
Fix can be done in specs-store.js in the desktop-gui package:
Line 42:
```
      const filesByDirectory = _.map(specs, (spec) => {
        // change \\ from Windows to /
        let name = spec.name.replace(/\\/g, '/');
        return name.split('/');
      })
```

**Comment by brian-mann** [comment]
Released in [`2.0.3`](https://docs.cypress.io/guides/references/changelog.html#2-0-3).
